package.json

```diff
@@ -25,16 +26,17 @@
   "author": "Eric Fischer",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/efischer19/reactifex/issues"
+    "url": "https://github.com/edx/reactifex/issues"
   },
-  "homepage": "https://github.com/efischer19/reactifex#readme",
+  "homepage": "https://github.com/edx/reactifex#readme",
   "devDependencies": {
     "eslint": "^7.32.0",
     "eslint-config-airbnb": "^18.2.1",
     "eslint-plugin-import": "^2.24.2",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.25.1",
-    "eslint-plugin-react-hooks": "^1.7.0"
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "semantic-release": "^17.0.0"
```

Comment:
Rather than include `semantic-release` as a dependency, our other libraries tend to just use a Github Action to do this.  See frontend-build's version here: https://github.com/edx/frontend-build/blob/master/.github/workflows/release.yml#L29